PizzaAPI/PizzaAPI/Controllers/CommentsController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using PizzaAPI.Models;
+using PizzaAPI.Repositories;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using PizzaAPI.Helpers;
+
+namespace PizzaAPI.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class CommentsController : ControllerBase
+    {
+        private readonly ICommentsRepository _repository;
+
+        public CommentsController(ICommentsRepository repository)
+        {
+            _repository = repository;
+        }
+
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Comments>>> GetPizzas()
+        {
+            var pizzas = await _repository.GetAll();
+            return Ok(pizzas);
+        }
+
+        [HttpPost]
+        public void AddSuggestedPizza(Comments comments)
+        {
+            var upperComment = comments.Comment;
+            comments.Comment = upperComment.UppercaseFirst();
```

Comment:
```suggestion
            comments.Comment = comments.Comment.UppercaseFirst();
```
Szkoda robić tutaj dodatkową zmienną z którą nic nie robisz